### PR TITLE
🛠️🔐 `Security`: Review Policy objects and constrain default access levels

### PR DESCRIPTION
**Pull request by anaulin**
Working on https://github.com/zinc-collective/convene/pull/1778, I noticed that we have a few Scopes that are defined as `scope.all` for all cases, and probably shouldn't be.

Some of these scopes might not be in use at the moment, but I figured I could fix them all up to prevent future issues.

### CODE REVIEW

File: app/policies/person_policy.rb [hunk 1]
```diff
@@ -19,7 +19,13 @@ def permitted_attributes(_params)
 
   class Scope < ApplicationScope
     def resolve
-      scope.all
+      return scope.none if person.blank?
+
+      if person.operator?
+        scope.all
+      else
+        scope.where(id: person.id)
```
Comment (zspencer) [inline app/policies/person_policy.rb:27]: So I can only see myself?

**Review by zspencer — APPROVED** [review]

File: app/policies/person_policy.rb (hunk 1 above)
Comment (anaulin) [reply app/policies/person_policy.rb:27]: Yep! But you can see the `Memberships` for all of your Spaces (defined in the `MembershipPolicy`).

Comment (anaulin) [reply app/policies/person_policy.rb:27]: (Also, the `show?` policy method above matches this: you can only `show?` yourself if you're not an Operator.)

Comment (zspencer) [reply app/policies/person_policy.rb:27]: Makes sense! Thank you!

**Pull request merged**